test/smoke/package/resources/HermesReactNativeSample/build.gradle

```diff
@@ -114,16 +114,19 @@ def jscFlavor = 'org.webkit:android-jsc:+'
 /**
  * Whether to enable the Hermes VM.
  *
- * This should be set on project.ext.react and mirrored here.  If it is not set
+ * This should be set on project.ext.react and that value will be read here. If it is not set
  * on project.ext.react, JavaScript will not be compiled to Hermes Bytecode
  * and the benefits of using Hermes will therefore be sharply reduced.
  */
 def enableHermes = project.ext.react.get("enableHermes", false);
 
 /**
- * Architectures to build native code for in debug.
+ * Architectures to build native code for.
```

Comment:
The sentence is not finished.

Reply:
This file has been completely copied from a clean react-native v0.68.0 project. I think there is no need to correct this comment in this case.